backend/config/settings/base.py

```diff
@@ -120,3 +121,16 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+# List of sites from which you can make requests to this project.
+# To enable CORS_ALLOWED_ORIGINS, change CORS_ALLOW_ALL_ORIGINS to False
+CORS_ALLOWED_ORIGINS = [
+    'http://127.0.0.1:8000',
+
+]
+
+CORS_ALLOW_ALL_ORIGINS = True 
```

Comment:
Maybe comment out CORS_ALLOWED_ORIGINS if it is not yet used? Until CORS_ALLOW_ALL_ORIGINS is True, it will have effect anyway.

Reply:
It makes no sense, it's a regular switch. We don't use a lot of functionality yet)